### ISSUE TITLE
Request: Emoji

### ISSUE THREAD

**Issue by rvlouie** [ionic-team/ionicons#378]
🤓 🤖 🙃  <--- to do this, but you know, in a web font.

**Comment by The-Penultimate-Defenestrator** [comment]
Why don't you give some existing web fonts like [this](https://github.com/MorbZ/OpenSansEmoji) a look?

**Comment by christian-bromann** [comment]
I agree with @controversial , there are good alternatives for this type of icons.
